### DISCUSSION TITLE
Is there a search method ?

### DISCUSSION THREAD

**Discussion by KillianGrd35** [singerdmx/flutter-quill#742] [General]
Is there a search method where you can highlight (or underline) the matched words without modifying the document itself ?

Thanks in advance.

**Comment by singerdmx** [comment]
Yeah, we probably need to add this functionality

**Comment by singerdmx** [comment]
Planning to add this. I am not a good PM so any suggestion regarding UI and the workflow is welcome

**Comment by singerdmx** [comment]
I think we should expose a public function that users can call, which takes in the string and return a list of offset that matches the string. Also I think we already have function to move cursor to target offset. So it is up to users to do whatever with the UI.
So UI will not be part of the implementation.

**Comment by singerdmx** [comment]
I have submitted https://github.com/singerdmx/flutter-quill/commit/d2b308dc4d4c70620991185bbd1d0dd0911b69a5

**Comment by singerdmx** [comment]
Search feature released in 5.3.0
